app/views/lab_spaces/show.html.erb

```diff
@@ -316,35 +316,34 @@
         imgFile = image[0].file;
       }
       let formData = {
-        name: name,
-        description: desc,
-        technical_description: techdesc,
-        capabilities: capacidades,
-        materials: materiales,
-        image: imgFile
+        equipment: {
+          name: name,
+          description: desc,
+          technical_description: techdesc,
+          capability_ids: capacidades,
+          material_ids: materiales,
+          image: imgFile,
+          available_hours: []
+        },
+        authenticity_token: $('meta[name="csrf-token"]').attr('content')
       };
-      let fd = new FormData();
-      for(var key in formData) {
-        fd.append(`equipment[${key}]`, formData[key]);
-      }
       for (var i = 0; i < availSchedule.length; i++) {
         if(availSchedule[i] === undefined) {
           continue;
         }
-        fd.append(`equipment[available_hours][${i}][day_of_week]`, availSchedule[i]['day_of_week']);
-        fd.append(`equipment[available_hours][${i}][start_time]`, availSchedule[i]['start_time'] + ' CT');
-        fd.append(`equipment[available_hours][${i}][end_time]`, availSchedule[i]['end_time'] + ' CT');
+        formData.equipment.available_hours.push({
+          day_of_week: availSchedule[i]['day_of_week'],
+          start_time: availSchedule[i]['start_time'] + ' CT',
+          end_time: availSchedule[i]['end_time'] + ' CT'
+        });
       }
-      fd.append("authenticity_token", $('meta[name=csrf-token]').attr('content'));
 
       // Get horarios
       // Get photo
       $.ajax({
         url: "<%= lab_lab_space_equipment_index_path lab_id: @lab_space.lab.id, lab_space_id: @lab_space.id %>", // TODO: change this!
-        contentType: false,
-        processData: false,
         cache: false,
-        data: fd,
+        data: formData,
```

Comment:
Does this work? :O lo estaba mandando a traves de un form data porque es multipart

Reply:
sii, lo cambié porque FormData estaba cambiando arrays a strings y quería el array

Reply:
Y si te llega bien la imagen?

Reply:
ahhh, tienes razón, fue lo único que no probé, no jala

Reply:
ya lo revertí